bezier-rs/docs/interactive-docs/src/utils/drawing.ts

```diff
@@ -95,27 +109,24 @@ export const drawBezier = (ctx: CanvasRenderingContext2D, points: Point[], dragI
 		handleStart = points[1];
 		handleEnd = points[2];
 		end = points[3];
-	} else {
+	} else if (points.length === 3) {
 		handleStart = points[1];
 		handleEnd = handleStart;
 		end = points[2];
-	}
-
-	ctx.strokeStyle = styleConfig.curveStrokeColor;
-	ctx.lineWidth = 2;
-
-	ctx.beginPath();
-	ctx.moveTo(points[0].x, points[0].y);
-	if (points.length === 3) {
-		ctx.quadraticCurveTo(handleStart.x, handleStart.y, end.x, end.y);
 	} else {
-		ctx.bezierCurveTo(handleStart.x, handleStart.y, handleEnd.x, handleEnd.y, end.x, end.y);
+		handleStart = start;
+		handleEnd = points[1];
+		end = handleEnd;
 	}
-	ctx.stroke();
 
-	if (styleConfig.drawHandles) {
-		drawLine(ctx, start, handleStart, styleConfig.handleLineStrokeColor);
-		drawLine(ctx, end, handleEnd, styleConfig.handleLineStrokeColor);
+	if (points.length === 2) {
+		drawLine(ctx, start, end, styleConfig.curveStrokeColor);
```

Comment:
We should adjust the `linewidth` so that the thickness of the linear segment matches the cubic and quadratic beziers.